packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -1460,49 +1473,55 @@ export class IntervalCollection
 			throw new LoggingError("attachSequence must be called");
 		}
 
-		const { localSeq } = localOpMetadata;
-		const { intervalType, properties, stickiness, startSide, endSide } = original;
+		const { localSeq, interval } = localOpMetadata;
 		const { id } = getSerializedProperties(original);
-		const { start: startRebased, end: endRebased } = (localOpMetadata.rebased ??=
-			this.computeRebasedPositions(localOpMetadata, squash));
-
+		const rebasedEndpoint = (localOpMetadata.rebased ??= this.computeRebasedPositions(
+			localOpMetadata,
+			squash,
+		));
 		const localInterval = this.localCollection.idIntervalIndex.getIntervalById(id);
 
-		const rebased: SerializedIntervalDelta = {
-			start: startRebased,
-			end: endRebased,
-			intervalType,
-			sequenceNumber: this.client?.getCurrentSeq() ?? 0,
-			properties,
-			stickiness,
-			startSide,
-			endSide,
-		};
-
 		// if the interval slid off the string, rebase the op to be a noop and delete the interval.
-		if (
-			!this.options.mergeTreeReferencesCanSlideToEndpoint &&
-			(startRebased === DetachedReferencePosition || endRebased === DetachedReferencePosition)
-		) {
-			if (localInterval) {
+		if (rebasedEndpoint === "detached") {
+			if (
+				localInterval !== undefined &&
+				(localInterval === interval || localOpMetadata.type === "add")
```

Comment:
pre-existing, but we should probably be emitting a remove event here

Reply:
yeah. there are a lots of these :(